### DISCUSSION TITLE
Can it be made easier to update a custom fork of the insiders version?

### DISCUSSION THREAD

**Discussion by DanielHabenicht** [squidfunk/mkdocs-material#6586] [Q&A]
I was setting up my custom fork to keep access to past versions as I, unfortunately, can't fund monthly access for just my hobby sites. :( 
During this task I came across having to manually change the `build.yml` in these places: 

https://github.com/squidfunk/mkdocs-material-insiders/blob/870d69ad0f9bd599c3da77bd5327c22ab9993d34/.github/workflows/build.yml#L69

https://github.com/squidfunk/mkdocs-material-insiders/blob/870d69ad0f9bd599c3da77bd5327c22ab9993d34/.github/workflows/build.yml#L126

This for sure makes it easier to set up a fork via the GitHub UI but does not work when manually creating the fork to keep access after not sponsoring anymore. 

I would propose changing it to `if: github.repository != "squidfunk/mkdocs-material-insiders"` so it works for forks made via Github UI and custom forks.

No problem if it's not changed to keep people sponsoring more. :)
Thanks and keep up this amazing project!

**Comment by kamilkrzyskow** [comment]
Hello @DanielHabenicht,
I'm not entirely sure why do you need the `build.yml` workflow on your private repository copy to work. Are you making changes that require a rebuild? The red ❌ doesn't mean it's not installable via `pip git+...`.

**Reply by DanielHabenicht** [reply]
Hi thanks for your help. 
I am following the `Docker` Setup guide: https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-docker which needs to run the build pipeline.

**Reply by kamilkrzyskow** [reply]
OK, the docker part is a requirement. I didn't understand that at first.
The workflow condition expects a fork, and you create a hard fork disconnected from the insiders repository to prevent automatic deletion of the fork.
So create your own private organization, create the hard copy fork in that organization and then fork that repository in your personal account. 
Any future updates you should be able to push into that hard copy, and no changes to the build should be necessary.

Of course, changing a few lines in the workflow file would allow users to not deal with the shenanigans.

**Reply by DanielHabenicht** [reply]
Yes, it's also possible to just merge and apply the commit again on top. I just wanted to let somebody know that this is causing "unnecessary" friction. 
If you are fine with theses changes I am happy to send you a PR.
